src/components/Home.vue

```diff
@@ -200,6 +202,14 @@ export default {
             for (const addr in this.coldAddresses) {
                 this.getBalance(addr)
             }
+            for (var i in this.coldAddresses) {
+                if (!this.coldAddresses[i].hasOwnProperty('api')) {
+                    let tempObj = {'protocol': 'v.systems', 'opc': 'account', 'address': i, 'api': 1, 'publicKey': this.coldAddresses[i]}
+                    Vue.set(this.coldAddresses, i, JSON.parse(JSON.stringify(tempObj)))
+                    this.setUsrLocalStorage('coldAddresses', JSON.stringify(this.coldAddresses))
```

Comment:
It's not a good idea to reset the whole ```coldAddresses``` in local storage in a loop.
The local storage should be set after the loop